src/data/setoid.lean

```diff
@@ -22,9 +22,11 @@ set - along with some preliminary results about `bell`.
 The function `rel` and lemmas ending in ' make it easier to talk about different 
 equivalence relations on the same type.
 
-The complete lattice instance for setoids could be defined by lifting the Galois
-insertion of equivalence relations on α into binary relations on α, but the resulting
-functions are less easy to use. 
+The complete lattice instance for equivalence relations could have been defined by lifting 
+the Galois insertion of equivalence relations on α into binary relations on α, and then using
+`complete_lattice.copy` to define a complete lattice instance with more appropriate 
+definitional equalities (a similar example is at line 284 of `order/filter/basic.lean`). 
```

Comment:
I think it would be better to (at least, also) refer to the name of the declaration. Line numbers are very unstable. I think the name is `filter.lattice.complete_lattice`. This name likely won't change.